packages/core/src/components/toast/_toast.scss

```diff
@@ -228,8 +228,18 @@ $toast-margin: $pt-grid-size * 1.5;
     background-color: $dark-gray5;
   }
 
-  &[class*="pt-intent-"] > .pt-icon-standard {
-    color: $white;
+  &[class*="pt-intent-"] {
```

Comment:
Just to verify - is `$white` the correct text color for both light theme and dark theme?